presto-spark-common/src/main/java/com/facebook/presto/spark/PrestoSparkQueryResults.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.spark;
+
+import com.facebook.presto.client.Column;
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.google.common.collect.ImmutableList;
+
+import java.util.List;
+
+import static java.util.Objects.requireNonNull;
+
+public class PrestoSparkQueryResults
+{
+    private final List<Column> columns;
+    private final List<List<Object>> data;
+
+    @JsonCreator
+    public PrestoSparkQueryResults(
```

Comment:
Note `QueryResults` in Presto classic implements two interface: `QueryData` and `QueryStatusInfo`. This seems more like `QueryData`, so maybe `PrestoSparkQueryData`?